src/app/api/github/webhook/route.ts

```
@@ -0,0 +1,80 @@
+                audit_reason: analysis.flagReason,
+                commit_count: analysis.totalCommits,
+                lines_changed: analysis.totalLinesChanged,
+            })
+            .eq('user_id', user.id)
+            .eq('series_id', user.series_id);
+
+        if (updateError) {
+            throw new Error(updateError.message);
+        }
+
+        // 4. If suspicious, trigger alert to Organizer (mocked)
```

ESLint

[error] Delete `····`

(prettier/prettier)